apm-sniffer/config/agent.config

```diff
@@ -61,4 +61,7 @@ logging.level=${SW_LOGGING_LEVEL:DEBUG}
 # logging.max_history_files=${SW_LOGGING_MAX_HISTORY_FILES:-1}
 
 # mysql plugin configuration
-# plugin.mysql.trace_sql_parameters=${SW_MYSQL_TRACE_SQL_PARAMETERS:false}
\ No newline at end of file
+# plugin.mysql.trace_sql_parameters=${SW_MYSQL_TRACE_SQL_PARAMETERS:false}
+
+# Jdk Threading configuration
+plugin.jdkthreading.threading_class_prefixes=${SW_JDK_THREADING_CLASS_PREFIXES:""}
```

Comment:
I think as an optional plugin, this is not recommended to add here.

Reply:
> I think as an optional plugin, this is not recommended to add here.

I was thinking that since it’s not activated by default, it does no harm to put the configuration here, I can comment it out if needed

Reply:
But we have dozens of configs, if we don't this here, why don't we put others. I think currently, the principle is putting minimal config here, to ease the first when they are new to SW.

Reply:
> But we have dozens of configs if we put this here, why don't we put others. I think currently, the principle is putting minimal config here, to ease the first when they are new to SW.

Make sense to me